src/Temporalio/Converters/FailureConverter.cs

```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Temporalio.Api.Failure.V1;
+using Temporalio.Exceptions;
+
+namespace Temporalio.Converters
+{
+    /// <summary>
+    /// Default implementation of <see cref="IFailureConverter" />.
+    /// </summary>
+    public class FailureConverter : IFailureConverter
```

Comment:
I don't think so. I only used `Default` in those others because of the name clash with the abstract/interface. In .NET the default impl of an interface being the interface name sans leading `I` makes the most sense.

Reply:
After some more thought, I think I will rename these

Reply:
Fixed